README.md

```diff
@@ -15,6 +15,11 @@ Ansible role for 7.x/6.x Elasticsearch.  Currently this works on Debian and RedH
 
 The latest Elasticsearch versions of 7.x & 6.x are actively tested.  **Only Ansible versions > 2.4.3.0 are supported, as this is currently the only version tested.**
 
+**BREAKING CHANGE**
+
+Starting with ansible-elasticsearch:7.0.0, installing more than one instance of Elasticsearch **on the same host** in no more supported.
```

Comment:
"in no more supported"

"is no longer supported"

Reply:
It seems that I make this typo really often 🤦‍♂

Reply:
Haha. It's still there btw :P.